src/ByteSizeLib/ByteSize.cs

```diff
@@ -142,37 +146,72 @@ public ByteSize(double bytes)
         }
 
+        /// <inheritdoc cref="FromBits" />
+        /// <param name="value">Number of bytes.</param>
         public static ByteSize FromBytes(double value)
         {
             return new ByteSize(value);
         }
 
         /// <summary>
-        /// Converts the value of the current object to a string.
-        /// The prefix symbol (bit, byte, kilo, mebi, gibi, tebi) used is the
+        /// Converts the value of the current object to a decimal byte string.
+        /// The prefix symbol (bit, byte, kilo, mega, etc.) used is the
         /// largest prefix such that the corresponding value is greater than or
         /// equal to one.
+        /// Use <see cref="ByteSize.ToBinaryString()"/> for binary string representation.
         /// </summary>
         public override string ToString()
         {
             return this.ToString("0.##", CultureInfo.CurrentCulture);
         }
 
+        /// <summary>
+        /// Converts the value of the current object to a decimal byte string.
+        /// The prefix symbol (bit, byte, kilo, mega, etc.) used is the
+        /// largest prefix such that the corresponding value is greater than or
+        /// equal to one.
+        /// Use <see cref="ByteSize.ToBinaryString()"/> for binary string representation.
+        /// </summary>
+        /// <param name="format">A numeric format string.</param>
         public string ToString(string format)
         {
             return this.ToString(format, CultureInfo.CurrentCulture);
         }
 
+        /// <summary>
+        /// Converts the value of the current object to a decimal byte string.
+        /// The prefix symbol (bit, byte, kilo, mega, etc.) used is the
+        /// largest prefix such that the corresponding value is greater than or
+        /// equal to one.
+        /// Use <see cref="ByteSize.ToBinaryString()"/> for binary string representation.
+        /// </summary>
+        /// <param name="format">A numeric format string.</param>
+        /// <param name="provider">An object that supplies culture-specific formatting information.</param>
         public string ToString(string? format, IFormatProvider? provider)
         {
             return this.ToString(format, provider, useBinaryByte: false);
         }
 
+        /// <summary>
+        /// Converts the value of the current object to a decimal byte string.
+        /// The prefix symbol (bit, byte, kilo, mega, etc.) used is the
+        /// largest prefix such that the corresponding value is greater than or
+        /// equal to one.
+        /// Use <see cref="ByteSize.ToBinaryString()"/> for binary string representation.
+        /// </summary>
+        /// <param name="format">A numeric format string.</param>
+        /// <param name="provider">An object that supplies culture-specific formatting information.</param>
+        /// <param name="useBinaryByte">Set to True to use binary byte values (1 KiB = 1024) instead of decimal values (1 KB = 1000 B).</param>
```

Comment:
When using `True` consider using `<see langword`:
```suggestion
        /// <param name="useBinaryByte">Set to <see langword="true"/> to use binary byte values (1 KiB = 1024) instead of decimal values (1 KB = 1000 B).</param>
```

Reply:
I think the "Set to..." part is a bit weird here and indicates some sort of "assignment", which is not a great fit for parameters IMHO.

This is obviously small and I don't have a very objective suggestion besides just using a different term, or outright removing it (like "True to use binary....").